src/components/form/date-input/index.tsx

```diff
@@ -115,34 +115,46 @@ export const DateInput = React.forwardRef<HTMLDivElement, DateInputProps>(
 
     return (
       <StyledDayPicker
-        className={cx('dateInput', !isValid && 'error', className)}
+        className={cx('dateInput', (!isValid || error) && 'error', className)}
         theme={theme}
         ref={ref}
         {...wrapperProps}
       >
-        {inputs.map((input, index) => {
-          const inputWidth = input.format.length * 10 + 40;
-
-          const isLastItem = index + 1 !== inputs.length;
-          return (
-            <div className='dateInputItem' key={`text-input-${input.type}`}>
-              <TextInput
-                id={`text-input-${input.type}`}
-                key={`text-input-${input.type}-input`}
-                onChange={(e): void => {
-                  handleChange(e, index, input.type);
-                }}
-                disabled={disabled}
-                inputRef={inputToRef[index]}
-                title={input.title}
-                style={{ width: `${inputWidth}px` }}
-                placeholder={input.placeholder}
-                value={currentDate.split('/')[index]}
-              />
-              {isLastItem ? <Text className='dateInputItemDivider'>{divider}</Text> : null}
-            </div>
-          );
-        })}
+        <div className='dateInputWrapper'>
```

Comment:
I'd use the MUI way for class names here. Like `PuiDayPicker-wrapper`

Reply:
In this way, there should not be any conflicts of names, because it's kind of a namespace that we use. Moreover, it's 100% clear which component we are styling or applying styles for.

Reply:
It would cause breaking changes. I like this approach, but i'd rather do this in next update

Reply:
ok